Changes to the main README.

dedan opened the pull request:
* I moved the detailed data part to a sub-README in notebooks.
* The detailed README for the frontend folder is still missing.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/bayesimpact/bob-emploi/4)
<!-- Reviewable:end -->

dedan commented:
+@pcorpet 

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-:-KaN1wD9wWdUYt4fj60d:bv2wshd)*
<!-- Sent from Reviewable.io -->

pcorpet commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[README.md, line 22 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2n3o3hGsjYm0rJxS:-KaN2n3o3hGsjYm0rJxT:blqdcfh) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L22)):*
> ```Markdown
> This repository contains several components where each is located within a single folder on the root level. Currently existing folders are:
> 
> * `frontend`: Containing the Bob Emploi web application.
> ```

Please drop the "Containing".

---

*[README.md, line 23 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2gDdOAF4MydRQyBH:-KaN2gDdOAF4MydRQyBI:bift9rp) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L23)):*
> ```Markdown
> 
> * `frontend`: Containing the Bob Emploi web application.
> * `notebooks`: Data analysis notebooks.
> ```

I'm not sure that a notebook is such a common thing or that one would know what it means in this context.

---

*[README.md, line 29 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2s00pp-n5ILWlBNw:-KaN2s00pp-n5ILWlBNx:b-wgf02l) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L29)):*
> ```Markdown
> ### Contributions
> 
> If you want to contribute to Bob Emploi with a bug fix or feature suggestion, please follow the Github contribution workflow described in [here](https://guides.github.com/activities/contributing-to-open-source/#contributing). At the current stage however, the main intention of the open source repository is to be transparent about the inner workings of Bob. In the future we hope  for, and will actively seek out for, contributions from the community. When we are ready for that, we will add detailed contribution instructions, suggestions of what to work on and a [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) to this repository.
> ```

Please move that to CONTRIBUTING.md (see https://github.com/blog/1184-contributing-guidelines).

---

*[README.md, line 54 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3L5NbJCLRKGwGP85:-KaN3L5NbJCLRKGwGP86:b-ar66tq) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L54)):*
> ```Markdown
> We have opened some of the data analysis notebooks (folder `notebooks`) pertaining to the data sources used by the application. These are unedited and incomplete, but provide a first insight on the data our recommendations are based on. As these are critical to being able to assess the validity of Bob Emploi's recommendation, we will first focus our open-sourcing efforts on opening more notebooks.
> 
> In particular, some of our datasets contain private data, or belong to an external data partner), and as such notebooks analyzing them will be opened over time as we're able to individually vet them for any potential leakage of sensitive information. The [notebooks README](notebooks/README.md) contains a list of the main data sources used in the application along with a short description.
> ```

error in ")", either drop it or add a preceding "("

---

*[README.md, line 54 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3SE0Zlhp5b1Lcn1X:-KaN3SE0Zlhp5b1Lcn1Y:b-45ar1k) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L54)):*
> ```Markdown
> We have opened some of the data analysis notebooks (folder `notebooks`) pertaining to the data sources used by the application. These are unedited and incomplete, but provide a first insight on the data our recommendations are based on. As these are critical to being able to assess the validity of Bob Emploi's recommendation, we will first focus our open-sourcing efforts on opening more notebooks.
> 
> In particular, some of our datasets contain private data, or belong to an external data partner), and as such notebooks analyzing them will be opened over time as we're able to individually vet them for any potential leakage of sensitive information. The [notebooks README](notebooks/README.md) contains a list of the main data sources used in the application along with a short description.
> ```

we're -> we are

---

*[notebooks/README.md, line 3 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3bnanQgwHbkotBHD:-KaN3bnanQgwHbkotBHE:bwo2irz) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/notebooks/README.md#L3)):*
> ```Markdown
> ### Data
> 
> We are currently veting and cleaning our data sources for open sourcing and plan to finish this process until the end of January 2017. As soon as we are done we will add detailed instructions of how to run the notebooks to this folder of the repository.
> ```

I'm not sure that "until" can be found after "plan to finish". Maybe "before" instead?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-:-KaN3nMixma7d355Xi7o:bnfp4nl)*
<!-- Sent from Reviewable.io -->

dedan commented:
Review status: all files reviewed at latest revision, 6 unresolved discussions, all commit checks successful.

---

*[README.md, line 22 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2n3o3hGsjYm0rJxS:-KaNM28dc3EmIezO44F9:b-896fix) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L22)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

Please drop the "Containing".
</blockquote></details>

Done.

---

*[README.md, line 23 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2gDdOAF4MydRQyBH:-KaNMMJq9jtrVbVHflRK:bgvwfkg) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L23)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

I'm not sure that a notebook is such a common thing or that one would know what it means in this context.
</blockquote></details>

Done. Changed the wording and added a link.

---

*[README.md, line 29 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN2s00pp-n5ILWlBNw:-KaNMbpV39Fzv0uuVcWC:b24k171) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L29)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

Please move that to CONTRIBUTING.md (see https://github.com/blog/1184-contributing-guidelines).
</blockquote></details>

I thought about that but don't think that the current text fits in there. `CONTRIBUTING.md` contains detailed information about how to test and format a PR and Github takes care to hint at it when a new PR is created. This paragraph however has the purpose to tell people about our intend of open sourcing and that we don't really expect contributions at the moment.

I already have a TODO to later add a detailed `CONTRIBUTING.md`.

---

*[README.md, line 54 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3L5NbJCLRKGwGP85:-KaNNIOVal2puXbs3dwy:b-896fix) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L54)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

error in ")", either drop it or add a preceding "("
</blockquote></details>

Done.

---

*[README.md, line 54 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3SE0Zlhp5b1Lcn1X:-KaNNLzCcBHJ9KBWXTtO:b-896fix) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/README.md#L54)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

we're -> we are
</blockquote></details>

Done.

---

*[notebooks/README.md, line 3 at r1](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4#-KaN3bnanQgwHbkotBHD:-KaNNTdvlBB1Ma9owK75:b-376vyb) ([raw file](https://github.com/bayesimpact/bob-emploi/blob/653fe86fc880cb8248b0adfeb6e6aac04ff41373/notebooks/README.md#L3)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

I'm not sure that "until" can be found after "plan to finish". Maybe "before" instead?
</blockquote></details>

Done. Good catch.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bayesimpact/bob-emploi/4)*
<!-- Sent from Reviewable.io -->
